src/main/java/com/ekart/config/SecurityConfig.java

```diff
@@ -72,9 +73,9 @@ public AuthenticationManager authenticationManagerBean() throws Exception {
     CorsConfigurationSource corsConfigurationSource() {
         CorsConfiguration configuration = new CorsConfiguration();
         configuration.setAllowedOrigins(Collections.singletonList("http://localhost:4200/"));
```

Comment:
remove the hardcoded origin part

Reply:
@prap12 , I tried using origin but it is not working..

![Screenshot 2021-12-22 at 9 03 16 PM](https://user-images.githubusercontent.com/79363607/147116965-b0ae1131-df36-4859-96e4-f34c6e3be4da.png)

also in request header, origin is present..

Reply:
okay will check